src/app/components/add-question/add-question.component.ts

```diff
@@ -48,68 +41,88 @@ export class AddQuestionComponent implements OnInit {
   }
 
   onAddQuestion(): void {
-    this.questions.push({
-      questionText: '',
-      hintText: '',
-      answers: [{ answerText: '', isCorrect: false }],
-    });
+    this.questionsArray.push(
+      this.fb.group({
+        id: [],
+        questionText: [, [Validators.required]],
+        hintText: [],
+        answers: this.fb.array([
```

Comment:
Do we need validation here that at least one answer should be correct?

Reply:
You can try add a custom validator here

Reply:
Created custom validator